myft/templates/save-for-later.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default ({ myFtApiWrite, contentId, variant, buttonText }) => {
```

Comment:
in general, arguments in an object are optional. looks like `contentId` is a required argument, so would right this as

```
(contentId, { myFtApiWrite, variant, buttonText })
```